pre-commit

```diff
@@ -42,6 +42,11 @@ if [[ -f "$git_root/.git" ]]; then
     fi
 fi
 
+# Only puppet 3.2.1 - 3.8 support "--parser future" option.
+case $(puppet --version) in
+  4*) USE_PUPPET_FUTURE_PARSER="disabled" ;;
```

Comment:
I did not add that feature, I just extended it to work in pre-commit hook aswell (previously it only worked in pre-receive hook); but I'll happyly add a short description in readme later today :)